### PR TITLE
desktop_branch PR

### PR DESCRIPTION
**Pull request by sibomana123**
For this project, I  used the knowledge gained in the previous week and I apply and used Flexbox CSS grid and media queries to make a responsive portfolio, this project seems good in all devices that have these sizes 375px, 640px, 768px, and above

### CODE REVIEW

File: desktopView.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  }
+
+  .description {
+    background-color: white;
+    background-image: url(img/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    border-bottom-left-radius: 6rem;
+    padding-bottom: 6rem;
+  }
```
Comment (joel-kalema) [inline desktopView.css]: - [ ] Good job working on the desktop version 🥇 , however, you need need to fix some parts, Kindly make sure to stick to the design as much as possible, I could be better if you fix your margins, font-size, and padding, let's take an example from this line of code:
````css
.description {
      .
      .
    height: 845px;
  }

  .header {
       .
    padding: 200px 150px 2px 200px;
    font-size: 48px;
       .
  }

  .parg {
       .
       .
    padding: 15px 100px 2px 200px;
    width: 90%;
       .
  }

  .let__go {
    padding: 1.2rem 100px 0 200px;
       .
  }

  .social__media {
       .
    padding: 12px 100px 0 200px;
  }
````

File: desktopView.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+  .social__media__img {
+    width: 13px;
+    color: #505f79;
+  }
+
+  /* ===================POST===================== */
+
+  .post__div {
+    display: flex;
```
Comment (joel-kalema) [inline desktopView.css]: - [ ] Kindly, make user you stack to the design (margins, colors, font-size,... ) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.) 😃. 

#### Figma design
Screenshot 2023-07-17 164849

#### Your design
Screenshot 2023-07-17 165148

File: index.html [hunk 3]
```diff
@@ -195,17 +209,32 @@ <h1>Languages</h1>
...
 
     </div>  
+  
   </div> 
- </section>   
-     
+
+  <div class="contact__container">
+   <form action="">
+    <h1>Contact Me</h1>
```
Comment (joel-kalema) [inline index.html]: - [ ] Also there, make sure to stick to the design 💪. Kindly, fix it 👍   

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |Screenshot 2023-07-17 172446| |Screenshot 2023-07-17 173830| |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @sibomana123 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

TN6


## To highlight :champagne: 

- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 


## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sibomana123** [comment]
After Review @joel-kalema  I tried to solve what he told me to change and used  font size style and color related to template

File: desktopVersion.css [hunk 4]
```diff
@@ -0,0 +1,502 @@
...
+  .post {
+    margin-top: 3rem;
+    background-color: #f1f1f1;
+    padding: 0 70px;
+  }
+
+  .post__container {
+    display: flex;
+    flex-direction: column;
+  }
```
Comment (Qoosim) [inline desktopVersion.css:127]: - [ ] Great job implementing required changes from the previous reviewer. However, there're few things to take care before you move on.
- [ ] Kindly ensure that you produce a close replica of the figma design at every section. I noticed that the width and the height of the project image is too much and as a result it affects how the project info should align beside the image. Kindly check the two screenshots and walk your design alongside the figma design. Pay attention to those areas I marked with red.

### Your Design
your-project-card

### Figma Design
temp-project=card

File: desktopVersion.css [hunk 5]
```diff
@@ -0,0 +1,502 @@
...
+    padding: 0 2px;
+    display: flex;
+    justify-content: center;
+    border-radius: 8px;
+    width: 135px;
+    height: 48px;
+    margin-top: 1.4rem;
+    background-color: transparent;
+    border: 1px solid #6070ff;
+  }
```
Comment (Qoosim) [inline desktopVersion.css]: - [OPTIONAL] Kindly make the text inside your button in a single line as it is on the figma design.

### Your Design
your-about

### Figma Design 
temp-about

File: desktopVersion.css [hunk 6]
```diff
@@ -0,0 +1,502 @@
...
+    background-color: white;
+    width: 120px;
+    height: 40px;
+    text-align: center;
+    margin-bottom: 2rem;
+    font-size: 17px;
+    outline: none;
+    border: none;
+  }
+}
```
Comment (Qoosim) [inline desktopVersion.css]: - [ ] As mentioned by the previous reviewer, you're required to stick to the figma design as close as possible. Your contact me section is quite different from the figma design. Kindly compare the two screenshots below and walk your design alongside the figma design

### Your Design
your-form

### Figam Design
temp-form

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @sibomana123, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  Desktop Version worked on :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

- [OPTIONAL] While trying to style your desktop version of your portfolio, you must ensure that it does not affect your mobile version. At this moment, your mobile version is greatly affected. Kindly look into that and fix.

### Mobile Version
[mobile.webm](https://github.com/sibomana123/MV__Portfolio/assets/26597432/58f75a8e-1bd6-4086-a245-2581642167c6)



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: desktopVersion.css [hunk 7]
```diff
@@ -0,0 +1,521 @@
...
+  .post {
+    margin-top: 3rem;
+    background-color: #f1f1f1;
+    padding: 0 50px;
+  }
+
+  .post__container {
+    display: flex;
+    flex-direction: column;
+  }
```
Comment (Alusp) [inline desktopVersion.css:127]: - [ ] Great job in implementing the changes requested by the previous reviewers, however, as mentioned by the previous reviewer, Kindly ensure that you produce a close replica of the Figma design in every section. I noticed that the width and the height of the project image are too much and as a result it affects how the project info should align beside the image.  matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Figma</td>
    </tr>
    <tr>
      <td>

 
Student de

 
</td>
      <td>

Figma De

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @sibomana123, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descriptive ✔️
- [ ] Professional GitHub Flow used ✔️
- [ ] Profesional PR Summary ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] While trying to style your desktop version of your portfolio, you must ensure that it does not affect your mobile version. At this moment, your mobile version is greatly affected. Kindly look into that and fix it.


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**